src/data/machines.js

```diff
@@ -46,22 +46,127 @@ export const gpuTypes = [
+  { name: 'EUROPE-WEST4-C', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-v100', 'nvidia-tesla-p4'] },
+  { name: 'EUROPE-WEST6-A', validTypes: [] },
+  { name: 'EUROPE-WEST6-B', validTypes: [] },
+  { name: 'EUROPE-WEST6-C', validTypes: [] },
+  { name: 'NORTHAMERICA-NORTHEAST1-A', validTypes: ['nvidia-tesla-p4'] },
+  { name: 'NORTHAMERICA-NORTHEAST1-B', validTypes: ['nvidia-tesla-p4'] },
+  { name: 'NORTHAMERICA-NORTHEAST1-C', validTypes: ['nvidia-tesla-p4'] },
+  { name: 'SOUTHAMERICA-EAST1-A', validTypes: [] },
+  { name: 'SOUTHAMERICA-EAST1-B', validTypes: [] },
+  { name: 'SOUTHAMERICA-EAST1-C', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'US-CENTRAL1-A', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-v100', 'nvidia-tesla-p4', 'nvidia-tesla-k80'] },
+  { name: 'US-CENTRAL1-B', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-v100'] },
+  { name: 'US-CENTRAL1-C', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-v100', 'nvidia-tesla-p100', 'nvidia-tesla-p4', 'nvidia-tesla-k80'] },
+  { name: 'US-CENTRAL1-F', validTypes: ['nvidia-tesla-v100', 'nvidia-tesla-p100'] },
+  { name: 'US-EAST1-B', validTypes: ['nvidia-tesla-p100'] },
+  { name: 'US-EAST1-C', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-v100', 'nvidia-tesla-p4', 'nvidia-tesla-k80'] },
+  { name: 'US-EAST1-D', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-k80'] },
+  { name: 'US-EAST4-A', validTypes: ['nvidia-tesla-p4'] },
+  { name: 'US-EAST4-B', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-p4'] },
+  { name: 'US-EAST4-C', validTypes: ['nvidia-tesla-p4'] },
+  { name: 'US-WEST1-A', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-v100', 'nvidia-tesla-p100'] },
+  { name: 'US-WEST1-B', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-v100', 'nvidia-tesla-p100', 'nvidia-tesla-k80'] },
+  { name: 'US-WEST1-C', validTypes: [] },
+  { name: 'US-WEST2-A', validTypes: [] },
+  { name: 'US-WEST2-B', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-p4'] },
+  { name: 'US-WEST2-C', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-p4'] },
+  { name: 'US-WEST3-A', validTypes: [] },
+  { name: 'US-WEST3-B', validTypes: [] },
+  { name: 'US-WEST3-C', validTypes: [] },
+  { name: 'US-WEST4-A', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'US-WEST4-B', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'US-WEST4-C', validTypes: [] }
 ]
 
 export const cloudServices = {
   GCE: 'GCE',
   DATAPROC: 'DATAPROC'
 }
 
-export const monthlyStoragePrice = 0.04 // from https://cloud.google.com/compute/pricing
-export const storagePrice = monthlyStoragePrice / 730 // per GB hour using 730 hours per month from https://cloud.google.com/compute/pricing
 export const dataprocCpuPrice = 0.01 // dataproc costs $0.01 per cpu per hour
 export const ephemeralExternalIpAddressPrice = { // per hour in dollars for Iowa (us-central1) region per https://cloud.google.com/vpc/network-pricing#ipaddress
   standard: 0.004,
   preemptible: 0.002
 }
 
+export const regionToDiskPrice = [
+  { name: 'ASIA-EAST1', monthlyDiskPrice: 0.04 },
```

Comment:
How will prices (here and above) be kept up-to-date with Google's pricing?

Reply:
Ideally, we would have some job running periodically that could update our database of prices. I have written a script that can pull these prices. Outside of this PR we could use this script in a cronjob somewhere. 

But I think for now we're fine to have the prices hard-coded here. They are not expected to change often at all. Prior to this PR the price of us-central1 resources were hard coded and haven't changed at all.